api/MediaKeySystemAccess.json

```diff
@@ -144,6 +144,251 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "audioCapabilities_property": {
```

Comment:
Since we don't have a guideline, maybe this is an opportunity to start from scratch? How about `returns_audioCapabilities_property` with the idea being that we could have a convention for object properties (like this one), types (like `returns_DOMMatrix_type`), and values (like `returns_undefined_value`).

We should probably put the word return in the description too. `Returns <code>audioCapabilities</code> property`?

Reply:
The phrasing "Returns <code>audioCapabilities</code> property" doesn't feel quite right to me. It's more like "Returned object has an <code>audioCapabilities</code> property", but that's a lot of words.

Reply:
Since the data is the same for all properties for Chromium and Safari, I decided it would be more reasonable to just create a note for Firefox, explaining two properties weren't available before Firefox 50.  If there were more entries or other browsers also had such variations, then keeping subfeatures seems reasonable, but since most of the data is the same as the parent I think a note is sufficient!

Reply:
More ideas on these lines, to cut down on the wordiness:

-  `audioCapabilities` in return value object
-  `audioCapabilities` in returned object
- Returns object with `audioCapabilities`
- Returns object with `audioCapabilities` property

Reply:
Ugh, sorry, I did not stop to think why I had the email open here. I ought to actually look at the change here. 🤦‍♂️

Reply:
All good Daniel!  I think it's a good convention to discuss and plan -- I've just opened #13834 so we may continue!